docs/source/driver.rst

```diff
@@ -144,6 +144,24 @@ The maximum time to allow for retries to be attempted when using transaction fun
 After this time, no more retries will be attempted.
 This setting does not terminate running queries.
 
+``resolver``
+------------
+
+A custom resolver function to use for DNS resolution.
```

Comment:
Maybe we can say that it is a hostname or IP address resolver and default implementation performs DNS resolution.

Reply:
:+1: Yes, that's better wording.